impeller/aiks/paint_pass_delegate.cc

```diff
@@ -72,7 +73,8 @@ bool OpacityPeepholePassDelegate::CanElide() {
 // |EntityPassDelgate|
 bool OpacityPeepholePassDelegate::CanCollapseIntoParentPass(
     EntityPass* entity_pass) {
-  if (paint_.color.alpha <= 0.0 || paint_.color.alpha >= 1.0 ||
+  if (paint_.blend_mode != BlendMode::kSourceOver ||
```

Comment:
This is handled in https://github.com/flutter/engine/blob/main/impeller/aiks/canvas.cc#L390

Reply:
Alrighty, removed this and threw in a comment.